db/people.yml

```diff
@@ -464,22 +464,40 @@ cbullard:
   active: true
   aka: []
   avatar: https://github.com/chloeann95.png
-  bio: bio
-  education: []
-  email: cab2344@columbia.edu
+  bio: Chloë grew up in Michigan and attended Michigan State University for her undergraduate degrees in Physics and Mechanical Engineering, where she worked on computational plasma physics research. Afterward she attended the University of Toledo completing a Masters in Chemical Engineering while working on modeling CFD simulations for biomass with NREL. When not working you can find her looking for a new game to play or a new song to learn on the piano.$
+
+  education:
+    - begin_year: 2013
+      begin_month: 9
+      degree: B.S. in Physics
+      degree: B.S. in Mechanical Engineering
+      institution: Michigan State University
+      end_year: 2018
+      end_month: 5
+    - begin_year: 2019
+      begin_month: 9
+      degree: M.S. in Chemical Engineering
+      institution: University of Toledo
+      end_year: 2021
+      end_month: 5
+    - begin_year: 2021
+      begin_month: 9
+      degree: M.S. in Material Science & Engineering
+      institution: Columbia University
+  email: chloe.bullard@columbia.edu
   employment:
     - begin_date: 2021-07-26
       end_date: 2022-12-31
       organization: columbiau
-      position: masters researcher
+      position: Masters Researcher
```

Comment:
This is ok.  If the lower case version renders all lower case, we should change the builder to use title-case.  It is better to have the app handle the casing in general.   Summary: leave this, but note to selves, let's check the website that all the positions appear correctly whatever case they have in the db.

Reply:
done, it does add capital for Masters but not researcher. I've always generally capitalized both as it tends to looks nicer in my opinion but can revert if preferred not.

Reply:
I think we found your first programming task!  I will create an issue on regolith and assign it to you.  Please look in the wiki and talk to the others about how to proceed.  Choose anyone to reach out to, whoever you feel most comfortable with, but if you have no preference Zach is a good place to start.